zeeguu/api/endpoints/user.py

```diff
@@ -88,6 +88,36 @@ def learned_and_native_language():
     res = dict(native=user.native_language_id, learned=user.learned_language_id)
     return json_result(res)
 
+@api.route("/get_user_unfinished_reading_sessions", methods=("GET",))
```

Comment:
the concept is `user_reading_session` so the name should maybe follow that? i.e., `get_unfinished_user_reading_sessions`?